util/utility.js

```diff
@@ -131,7 +131,7 @@ function buildMap() {
     */
 }
 
-const isRadiant = (p) => p.player_slot < 128;
+const isRadiant = (playerSlot) => playerSlot < 128;
```

Comment:
I mentioned before that the server side implementation of this function works on an object.  It's fine if you want to change it, but keep it mind it may confuse developers who work on both sides.